README.rst

```diff
@@ -174,7 +170,12 @@ To install XLoader:
 
    Ensure ``datastore`` is also listed, to enable CKAN DataStore.
 
-6. If it is a production server, you'll want to store jobs info in a more
+6. Starting CKAN 2.10 you will need to set an API Token to be able to
+   execute jobs against the server::
+
+     ckanext.xloader.api_token = <your-CKAN-generated-API-Token>
```

Comment:
this seems troublesome to auto configure.

Reply:
It also seems odd given that XLoader, as I recall, doesn't have to make API calls like DataPusher did. We'll have to look more into this before we upgrade to 2.10.